src/Microsoft.ML.HalLearners/VectorWhitening.cs

```diff
@@ -381,9 +381,12 @@ private static void GetColTypesAndIndex(IHostEnvironment env, IDataView inputDat
 
             for (int i = 0; i < columns.Length; i++)
             {
-                if (!inputSchema.TryGetColumnIndex(columns[i].Input, out cols[i]))
+                var col = inputSchema.GetColumnOrNull(columns[i].Input);
+                if (!col.HasValue)
                     throw env.ExceptSchemaMismatch(nameof(inputSchema), "input", columns[i].Input);
```

Comment:
>ExceptSchemaMismatch [](start = 30, length = 20)

This would be an example of a current  existing usage of `ExceptSchemaMismatch`, perhaps a pattern to follow in the other places where we are currently using `Except`.

Reply:
Cool!

---
In reply to: [246524124](https://github.com/dotnet/machinelearning/pull/2088#discussion_r246524124) [](ancestors = 246524124)